lib/libutee/include/user_ta_header.h

```diff
@@ -23,9 +23,13 @@
 	 * (pseudo-TAs only).
 	 */
 #define TA_FLAG_CONCURRENT		(1 << 8)
-#define TA_FLAG_DEVICE_ENUM		(1 << 9) /* device enumeration */
+	/*
+	 * Device enumeration.
```

Comment:
The need some more explanation. For instance:
_Device enumeration is done in two stages by normal world, first before the tee-supplicant has started and then once more when the tee-supplicant is started. The flags below controls if the TA should be reported in the first, second or in both cases._